index.html

```diff
@@ -24,55 +21,52 @@ <h1 id="title" class="title">
   </header>
-  <div>
+  <section class="form">
     <form id="survey-form">
       <div class="form-group">
-        <label id="name-label" for="name">Name</label>
-        <input id="name" name="name" type="text" required placeholder="Enter your name"> </input>
+        <label id="name-label" for="name">Name:</label>
+        <input class="input" id="name" name="name" type="text" required placeholder="Enter your name"> </input>
       </div>
       <div class="form-group">
-        <label for="email" id="email-label">Email</label>
-        <input id="email" name="email" type="email" required placeholder="Enter your email"> </input>
+        <label for="email" id="email-label">Email:</label>
+        <input class="input" id="email" name="email" type="email" required placeholder="Enter your email"> </input>
       </div>
       <div class="form-group">
         <label id="number-label" for="number">Age:</label>
-        <input id="number" name="number" min="18" max = "120" type="number" required placeholder="35"> </input>
+        <input class="input" id="number" name="number" min="18" max = "120" type="number" required placeholder="35"> </input>
       </div>
       <div class="form-group">
-        <label for="select">Dropdown</label>
-        <select name="age" id="dropdown"> 
+        <label for="select">Dropdown:</label>
+        <select class="input" name="age" id="dropdown"> 
           <option value="opt1">Option 1</option>
           <option value="opt2">Option 2</option>
         </select>
       </div>
       <div class="form-group">
-        <label>Gender</label>
-        <br>
-        <input type="radio" name="gender" value="male"> Male
-        <input type="radio" name="gender" value="female"> Female
-        <input type="radio" name="gender" value="other"> Other
+        <label>Gender:</label>
+        <div class="input flex-btw">
+          <input type="radio" name="gender" value="male"> Male </input>
+          <input type="radio" name="gender" value="female"> Female </input>
+          <input type="radio" name="gender" value="other"> Other </input>
+        </div>
       </div>
       <div class="form-group">
-        <label>Checkboxes</label> <br>
-        <label>
+        <label>Checkboxes:</label>
+        <div class="input flex-btw">
           <input name="check-opt" value="check1" type="checkbox">Check 1</input>
-        </label>
-        <label>
           <input name="check-opt" value="check2" type="checkbox">Check 2</input>
-        </label>
-        <label>
           <input name="check-opt" value="check3" type="checkbox">Check 3</input>
-        </label>
+        </div>
       </div>
       <div class="form-group">
-        <p><label for="comments">Additional Comments:</label></p>
-        <textarea id="w3review" name="w3review" rows="4" cols="50"></textarea>
+        <label for="comments">Additional Comments:</label>
+        <textarea class="input" id="w3review" name="w3review" rows="4" cols="50"></textarea>
```

Comment:
It is best practice to name your classes or id differently from the HTML tags or semantics